lib/drawer/index.js

```diff
@@ -32,13 +32,13 @@ export default (svg, dimensions, scales, configuration) => {
         .attr('clip-path', 'url(#drops-container-clipper)')
         .style('filter', 'url(#metaballs)');
 
-    const extremaContainer = svg.append('g')
+    svg.append('g')
         .classed('extremum', true)
         .attr('width', dimensions.width)
         .attr('height', 30)
         .attr('transform', `translate(${configuration.margin.left}, ${configuration.margin.top - 45})`);
 
-    configuration.metaballs && metaballs(defs);
+    configuration.metaballs && metaballs(defs); // eslint-disable-line no-unused-expressions
```

Comment:
Can we instead use: 

``` js
if (configuration.metaballs) {
    metaballs(defs);
}
```

It would prevent from an ESLint comment and would be clearer.